xarray/core/computation.py

```diff
@@ -1701,13 +1701,18 @@ def dot(
         Which dimensions to sum over. Ellipsis ('...') sums over all dimensions.
         If not specified, then all the common dimensions are summed over.
     **kwargs : dict
-        Additional keyword arguments passed to numpy.einsum or
-        dask.array.einsum
+        Additional keyword arguments passed to ``numpy.einsum`` or
+        ``dask.array.einsum``
 
     Returns
     -------
     DataArray
 
+    Notes
+    -----
+    We recommend either passing `optimize=True` or installing the optional
```

Comment:
I'm sure I'm missing something, but where does `optimize` get picked up?

Reply:
how about now?

Reply:
(I had meant where it was picked up in our code, I now see that it's passed through to numpy...)